dll/win32/shell32/folders/CRecycleBin.cpp

```diff
@@ -58,18 +58,57 @@ static const columninfo RecycleBinColumns[] =
 
 #define COLUMNS_COUNT  6
 
+BOOL WINAPI CBSearchRecycleBin(IN PVOID Context, IN HANDLE hDeletedFile);
+
+static PIDLRecycleStruct * _ILGetRecycleStruct(LPCITEMIDLIST pidl);
+
+typedef struct
+{
+    PIDLRecycleStruct *pFileDetails;
+    HANDLE hDeletedFile;
+    BOOL bFound;
+} SEARCH_CONTEXT, *PSEARCH_CONTEXT;
```

Comment:
Please move all that just after the "Recycle Bin folder" below.

Reply:
Done